components/mfa/setup/setup.tsx

```diff
@@ -117,7 +121,7 @@ export default class Setup extends React.PureComponent<Props, State> {
 
                 return;
             }
-
+            trackEvent('mfa_setup', 'mfa_setup_saved', {isMobile: isMobile()});
```

Comment:
@M-ZubairAhmed Do you know if there are any restrictions for the prop values of `trackEvent`? This is a boolean but every example in the webapp is either a number or a string